### PR TITLE
Updated documenation

### PR DESCRIPTION
**Pull request by marcin-krolik**
Updated documentation

Summary of changes:
- updated links
- updated example
- capitalized Snap

@intelsdi-x/plugin-maintainers

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -42,30 +42,28 @@ To install sysstat package from the official repositories simply use:
...
 ```
-This builds the plugin in `/build/rootfs/`
+This builds the plugin in `./build/`
 
 ### Configuration and Usage
-* Set up the [snap framework](https://github.com/intelsdi-x/snap/blob/master/README.md#getting-started)
-* Ensure `$SNAP_PATH` is exported  
-`export SNAP_PATH=$GOPATH/src/github.com/intelsdi-x/snap/build`
+* Set up the [Snap framework](https://github.com/intelsdi-x/snap#getting-started)
+* Load the plugin and create a task, see example in [Examples](https://github.com/intelsdi-x/snap-plugin-collector-iostat#examples).
```
Comment (kindermoumoute) [inline README.md]: Change URL to `#examples` to make it links correctly on different branch/fork.

File: README.md [hunk 2]
```diff
@@ -75,8 +73,8 @@ Customize path to iostat executable is also possible by setting environment vari
 To learn more about this plugin and iostat tool, visit:
 
 * [linux iostat tool] (http://linux.die.net/man/1/iostat)
-* [snap iostat unit test](https://github.com/intelsdi-x/snap-plugin-collector-iostat/blob/master/iostat/iostat_test.go)
-* [snap iostat examples](#examples)
+* [Snap iostat unit test](https://github.com/intelsdi-x/snap-plugin-collector-iostat/blob/master/iostat/iostat_test.go)
```
Comment (kindermoumoute) [inline README.md]: Change URL to `iostat/iostat_test.go` to make it links correctly on different branch/fork.

File: README.md (hunk 1 above)
Comment (marcin-krolik) [inline README.md]: done

File: README.md (hunk 2 above)
Comment (marcin-krolik) [inline README.md]: done

**Pull request merged**